frontend/src/components/elements/Progress/Progress.tsx

```diff
@@ -41,7 +41,9 @@ class Progress extends React.PureComponent<Props> {
     const isMovingBackwards = value < this.lastValue
     const isMovingSuperFast = time - this.lastAnimatedTime < FAST_UPDATE_MS
     const className =
-      isMovingBackwards || isMovingSuperFast
+      isMovingBackwards ||
+      isMovingSuperFast ||
+      document.visibilityState === "hidden"
```

Comment:
Just add a comment explaining why this check is here.

Reply:
Actually, having a variable like `isFoo` would be better

Reply:
Makes sense , thanks @kantuni